### ISSUE TITLE
Unexpected result for background-size shorthand

### ISSUE THREAD

**Issue by toptalo** [sass/libsass#1642]
The following code:

```scss
.some-block {
    background: transparent url("image.jpg") 50% 50% / cover no-repeat;
}
```
Results in the following with sass

```css
.some-block {
    background: transparent url("image.jpg") 50% 50% / cover no-repeat;
}
```

But the following with libsass:

```css
.some-block {
    background: transparent url("image.jpg") 50% 50%/"cover" no-repeat;
}
```

**Comment by xzyfer** [comment]
This was fixed in 3.3.1 which was released today. Please try again on that
version to see if the problem persists.

**Comment by toptalo** [comment]
Yes! Thanks!
